components/ActivityIndicator/examples/ActivityIndicatorExample.m

```diff
@@ -17,7 +17,10 @@
 #import <UIKit/UIKit.h>
 
 #import "ActivityIndicatorExampleSupplemental.h"
+
+#import "MDCActivityIndicatorColorThemer.h"
```

Comment:
lets not make the examples depend on the themer. (it doesn't look like this is needed anyways)